awx/ui_next/CONTRIBUTING.md

```diff
@@ -35,6 +35,7 @@ Have questions about this document or anything not covered here? Feel free to re
   - functions should adopt camelCase
   - constructors/classes should adopt PascalCase
   - constants to be exported should adopt UPPERCASE
+- For strings we adopt the `sentence capitalization` since it is a (patternfly style guide)[https://www.patternfly.org/v4/design-guidelines/content/grammar-and-terminology#capitalization].
```

Comment:
"For strings, we adopt `sentence capitalization` since it is part of the (patternfly style guide)"

Could make it like that ^ but nbd either way!